index.html

```diff
@@ -164,7 +164,7 @@ <h2 class="project-title">JavaScript Online BookList App</h2>
             <!------------------------------------About section------------------------------------>
             <section class="about-section">
                 <div class="about-header">
-                    <h2>About Me</h2>
+                    <h2>about me</h2>
```

Comment:
[Optional] I will suggest that you start with a capital letter as you have in your Figma design. 😁

Reply:
Hi Dear @Simpleshaikh1,

Thanks for your suggestion I will do this changes